web/src/features/map/Map.tsx

```diff
@@ -371,64 +361,14 @@ export default function MapPage({ onMapLoad }: MapPageProps): ReactElement {
       style={{ minWidth: '100vw', height: '100vh' }}
       mapStyle={MAP_STYLE as mapboxgl.Style}
     >
-      <Layer id="ocean" type="background" paint={styles.ocean} />
-
-      <Source
-        id="zones-clickable"
-        promoteId={'zoneId'}
-        type="geojson"
-        data={worldGeometries}
-      >
-        <Layer id="zones-clickable-layer" type="fill" paint={styles.zonesClickable} />
-        <Layer id="zones-hoverable-layer" type="fill" paint={styles.zonesHover} />
-        <Layer id="zones-border" type="line" paint={styles.zonesBorder} />
+      <Layer id="ocean" type="background" paint={mapStyles.ocean} />
+      <Source id={ZONE_SOURCE} promoteId={'zoneId'} type="geojson" data={worldGeometries}>
+        <Layer id="zones-clickable-layer" type="fill" paint={mapStyles.zonesClickable} />
+        <Layer id="zones-hoverable-layer" type="fill" paint={mapStyles.zonesHover} />
+        <Layer id="zones-border" type="line" paint={mapStyles.zonesBorder} />
       </Source>
```

Comment:
If you want to go further, you could attempt to extract the zone-related layers to it's own component - but it might not be that easy 😅

Reply:
Have moved this out too, what do ya think?